qfieldsync/gui/cloud_projects_dialog.py

```diff
@@ -403,7 +407,7 @@ def on_projects_cached_project_files_updated(self, project_id: str) -> None:
                     item.setData(0, Qt.UserRole, project_file)
 
                     item.setText(1, str(project_file.size))
-                    item.setTextAlignment(1, Qt.AlignRight)
+                    item.setTextAlignment(1, Qt.AlignmentFlag.AlignRight)
```

Comment:
I assume these are backwards compatible for Qt 5.15.x?

Reply:
@suricactus , correct.